src/api/clinical/medication/medication/medication.auth.ts

```diff
@@ -48,31 +48,31 @@ export class MedicationAuth {
         ...DefaultAuthOptions,
         Context     : `${this._baseContext}.GetAdministrationRoutes`,
         Ownership   : ResourceOwnership.System,
-        ActionScope : ActionScope.Public,
+        ActionScope : ActionScope.Tenant,
```

Comment:
This is public resource. Should be open for everybody. It basically lists types for giving medicine to the patient. This is common to all for all tenants.

Reply:
Yes, Done chnages.